### PR TITLE
[FIX] im_livechat: AttributeError: discuss.channel.livechat_visitor_id

### PR DESCRIPTION
**Pull request by jess-sys**
Description of the issue/feature this PR addresses:
Livechat is set up with a Chatbot with a fallback to an operator.
When a visitor visits a website with the Livechat and the visitor is not connected to an account, when falling back to the operator, an error occurs :
`AttributeError: 'discuss.channel' object has no attribute 'livechat_visitor_id'`

Current behavior before PR:
When a visitor visits a website with the Livechat and the visitor is not connected to an account, when falling back to the operator, an error occurs :
`AttributeError: 'discuss.channel' object has no attribute 'livechat_visitor_id'`

Desired behavior after PR is merged:
The chatbot falls back to the human operator.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by ThalusA** [comment]
Odoo Team, is it possible to provide further information about why Odoo Test Suite fail in such case ?

File: addons/im_livechat/models/chatbot_script_step.py [hunk 1]
```diff
@@ -330,8 +330,8 @@ def _process_step_forward_operator(self, discuss_channel):
 
         if discuss_channel.livechat_channel_id:
             human_operator = discuss_channel.livechat_channel_id._get_operator(
-                lang=discuss_channel.livechat_visitor_id.lang_id.code,
-                country_id=discuss_channel.country_id.id
+                lang=discuss_channel.livechat_visitor_id.lang_id.code if hasattr(discuss_channel, "livechat_visitor_id") else None,
```
Comment (alexkuhn) [inline addons/im_livechat/models/chatbot_script_step.py:333]: `livechat_visitor_id` is indeed only defined when the module `website_livechat` is installed, which requires to install `website` in addition to `im_livechat`: https://github.com/odoo/odoo/blob/17.0/addons/website_livechat/models/discuss_channel.py#L11

File: addons/im_livechat/models/chatbot_script_step.py [hunk 2]
```diff
@@ -330,8 +330,8 @@ def _process_step_forward_operator(self, discuss_channel):
...
+                country_id=discuss_channel.country_id.id if hasattr(discuss_channel, "country_id") else None
```
Comment (alexkuhn) [inline addons/im_livechat/models/chatbot_script_step.py:334]: I don't think we need to guard on `country_id`, as this field is defined in `im_livechat`: https://github.com/odoo/odoo/blob/17.0/addons/im_livechat/models/discuss_channel.py#L28

**Review by alexkuhn — COMMENTED** [review]
> Odoo Team, is it possible to provide further information about why Odoo Test Suite fail in such case ?

This was a false positive. I re-run the tests and they pass!

File: addons/im_livechat/models/chatbot_script_step.py (hunk 2 above)
Comment (jess-sys) [reply addons/im_livechat/models/chatbot_script_step.py]: Removed the guard on `country_id`.

**Review by alexkuhn — APPROVED** [review]
Thank you for the fix!

@robodoo r+ rebase-ff

**Pull request closed without merging**